core/server/models/base/token.js

```diff
@@ -36,7 +36,7 @@ Basetoken = ghostBookshelf.Model.extend({
             .query('where', 'expires', '<', Date.now())
             .fetch(options)
             .then(function then(collection) {
-                collection.invokeThen('destroy', options);
+                return collection.invokeThen('destroy', options);
```

Comment:
Looking at this code now, this seems really quite weird. What is the difference between `invokeThen` and calling `destroy()` itself? Just asking out of interest.

Also, I know that bookshelf is moving away from using collections entirely, so perhaps this code needs a different refactor? Again, wanted to flag this up whilst we're here so that we know what needs to be done